buildlogger/get.go

```diff
@@ -63,7 +62,7 @@ func (opts *BuildloggerGetOptions) parse() (string, error) {
 
 	params := fmt.Sprintf(
 		"?execution=%d&proc_name=%s&print_time=%v&print_priority=%v&n=%d&limit=%d&paginate=true",
-		opts.Execution,
+		opts.CedarOpts.Execution,
```

Comment:
Should this only be included if the execution is set? Or is execution=0 considered the latest execution?

Reply:
This always expects an execution, we can change it later but for what its used for right now this is fine.